openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -45,14 +45,6 @@ def groups(self):
             for mode in CourseMode.modes_for_course(course_key, include_expired=True, only_selectable=False)
         ]
 
-    def to_json(self):
-        """
-        Because this partition is dynamic, to_json and from_json are not supported.
-
-        Calling this method will raise a TypeError.
-        """
-        raise TypeError("Because EnrollmentTrackUserPartition is a dynamic partition, 'to_json' is not supported.")
-
     def from_json(self):
         """
         Because this partition is dynamic, to_json and from_json are not supported.
```

Comment:
Update comment-- to_json is now supported, but shouldn't be used to persist this partition within the course itself (used by Studio for sending data to front-end code).